tests/test_smoke_eval.py

```diff
@@ -56,7 +49,7 @@ def check_contains_files(dir: Path, expected_files: list[str]):
         assert file in created_file_names
 
 
-def eval_run(elicit: Elicit, tfr_datasets: list[str] = None) -> int:
+def eval_run(elicit: Elicit, transfer_datasets: Sequence[str] = []) -> int:
```

Comment:
btw its a sequence instead of a list cos list is mutable, so pyright will complain about mutable defaults. but with a protocol of sequence, pyright will complain if you try and mutate it, so that mutable default warning goes awa

Reply:
I see, thanks for explaining!!